kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -1119,6 +1121,13 @@ def explore_args(self) -> ArgumentParser:
             action='store_true',
             help='Store a node for every EVM basic block (implies --break-on-calls).',
         )
+        args.add_argument(
+            '--symbolic-immutables',
+            dest='break_on_load_program',
+            default=None,
+            action='store_true',
+            help='Enable support for symbolic immutables in Solidity constructor code.',
```

Comment:
```suggestion
            help='Enable support for symbolic parameters in Solidity constructor code.',
```
I understand it's a nit and `--symbolic-immutables` looks good as the argument name, but technically that enables any symbolic arguments being passed to constructor, not necessarily for immutables. Please feel free to ignore!

Reply:
Should I change to `--symbolic-constructor` instead?

Reply:
Yes, sounds great to me!